source/administration/command-line-tools.rst

```diff
@@ -1628,23 +1628,28 @@ mattermost user convert
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
   Description
-    Convert a user to a bot. If converting multiple users, use a space-separated list.
+    Convert a user to a bot, or convert a bot to a user account.
```

Comment:
Please help also update the description here https://github.com/mattermost/docs/blob/f6489b30bd994c33caaffa0cb658de592cab69b3/source/administration/command-line-tools.rst#mattermost-user.